src/main/java/com/sngular/kloadgen/sampler/schemaregistry/impl/ConfluentSchemaRegistry.java

```diff
@@ -0,0 +1,108 @@
+package com.sngular.kloadgen.sampler.schemaregistry.impl;
+
+import io.apicurio.registry.rest.client.RegistryClientFactory;
+import io.confluent.kafka.schemaregistry.avro.AvroSchemaProvider;
+import io.confluent.kafka.schemaregistry.client.CachedSchemaRegistryClient;
+import io.confluent.kafka.schemaregistry.client.SchemaMetadata;
+import io.confluent.kafka.schemaregistry.client.SchemaRegistryClient;
+import io.confluent.kafka.schemaregistry.client.rest.exceptions.RestClientException;
+import io.confluent.kafka.schemaregistry.json.JsonSchemaProvider;
+import io.confluent.kafka.schemaregistry.protobuf.ProtobufSchemaProvider;
+import io.confluent.kafka.serializers.AbstractKafkaSchemaSerDeConfig;
+import com.sngular.kloadgen.exception.KLoadGenException;
+import com.sngular.kloadgen.sampler.schemaregistry.SchemaRegistryConstants;
+import com.sngular.kloadgen.sampler.schemaregistry.SchemaRegistryManager;
+import com.sngular.kloadgen.util.SchemaRegistryKeyHelper;
+import org.apache.jmeter.threads.JMeterContextService;
+
+import java.io.IOException;
+import java.util.*;
+
+public class ConfluentSchemaRegistry implements SchemaRegistryManager {
+
+    private SchemaRegistryClient schemaRegistryClient;
+    private Map<String, String> propertiesMap;
+
+    public ConfluentSchemaRegistry() {
+        this.propertiesMap = new HashMap<>();
+        this.propertiesMap.put(SchemaRegistryKeyHelper.SCHEMA_REGISTRY_NAME, SchemaRegistryConstants.SCHEMA_REGISTRY_CONFLUENT);
+        this.propertiesMap.put(SchemaRegistryKeyHelper.SCHEMA_REGISTRY_URL_KEY, AbstractKafkaSchemaSerDeConfig.SCHEMA_REGISTRY_URL_CONFIG);
+        this.propertiesMap.put(SchemaRegistryConstants.BASIC_AUTH_CREDENTIALS, AbstractKafkaSchemaSerDeConfig.BASIC_AUTH_CREDENTIALS_SOURCE);
+        this.propertiesMap.put(SchemaRegistryConstants.USER_INFO_CONFIG, AbstractKafkaSchemaSerDeConfig.USER_INFO_CONFIG);
+        this.propertiesMap.put(SchemaRegistryConstants.BEARER_AUTH_CREDENTIALS , AbstractKafkaSchemaSerDeConfig.BEARER_AUTH_CREDENTIALS_SOURCE);
+        this.propertiesMap.put(SchemaRegistryConstants.BEARER_AUTH_TOKEN_CONFIG , AbstractKafkaSchemaSerDeConfig.BEARER_AUTH_TOKEN_CONFIG);
+    }
```

Comment:
Is this map meant to be modifiable on a per-instance basis? All of the values here are constants and the getter for it doesn't seem to be in use.

In case it doesn't need to be different on different instances it could be static; in case it doesn't need to be modifiable it could be immutable